src/main/webapp/js/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
 import App from './components/app';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
I like to use the following import order:
- node modules components
- blank line
- node modules css
- project css
- blank line
- project components

apply this to all of the files in this PR